### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by joon0615**
### Category
- [ ] New feature
- [ ] Bug fix
- [ ] Improvement
- [X] Refactor
- [ ] etc

### Description
- Add depth in group comparison feature

### CODE REVIEW

File: actions/src/main.py [hunk 1]
```diff
@@ -42,18 +55,37 @@ def deploy_to_group(github, group, init) -> None:
...
+        # if init:
+        #     workflows = _get_workflows('common')
+        # else:
+        #     workflows = _get_workflows(group)
+        #
+        # github._deploy(repo, workflows, init)
+        deploy_to_repository(github, repo_name, init)
+
+
+def _get_workflow_path(group, repo_name) -> str:
```
Comment (GeunjeLEE) [inline actions/src/main.py:69]: How about considering the default keyword arguments?
https://www.geeksforgeeks.org/default-arguments-in-python/

File: actions/src/main.py [hunk 2]
```diff
@@ -18,21 +19,33 @@ def main():
     else:
         sys.exit(1)
 
+
+def _format_repo_name(org_name: str, repo_name_full: str) -> str:
```
Comment (GeunjeLEE) [inline actions/src/main.py:23]: I think i'd better move to the bottom

File: actions/src/main.py [hunk 3]
```diff
@@ -42,18 +55,37 @@ def deploy_to_group(github, group, init) -> None:
     repo_names = _filter_match_repository_topics_to_group(group, all_repositories)
 
     for repo_name in  repo_names:
-        repo = github._get_repo(repo_name)
+        # repo = github._get_repo(repo_name)
```
[the remark this answers is not in the document]
Comment (joon0615) [reply actions/src/main.py]: Resolved. Deleted comment-out codes in main.py.

File: actions/src/main.py (hunk 2 above)
Comment (joon0615) [reply actions/src/main.py]: Resolved. Moved the function to the bottom where other private functions are implemented.

**Comment by GeunjeLEE** [comment]
LGTM!

**Pull request merged**